src/thorlabs_cube/driver/tcube/tpz.py

```diff
@@ -348,28 +350,28 @@ async def get_output_lut_parameters(self):
         get_msg = await self.send_request(
             MGMSG.PZ_REQ_OUTPUTLUTPARAMS,
             [MGMSG.PZ_GET_OUTPUTLUTPARAMS],
-            1,
+            _CHANNEL,
         )
         return st.unpack("<HHLLLL", get_msg.data[2:22])
 
-    async def start_lut_output(self):
+    async def start_lut_output(self) -> None:
         """Start the voltage waveform (LUT) outputs."""
-        await self.send(Message(MGMSG.PZ_START_LUTOUTPUT, param1=1))
+        await self.send(Message(MGMSG.PZ_START_LUTOUTPUT, param1=_CHANNEL))
 
-    async def stop_lut_output(self):
+    async def stop_lut_output(self) -> None:
         """Stop the voltage waveform (LUT) outputs."""
-        await self.send(Message(MGMSG.PZ_STOP_LUTOUTPUT, param1=1))
+        await self.send(Message(MGMSG.PZ_STOP_LUTOUTPUT, param1=_CHANNEL))
 
-    async def set_eeprom_parameters(self, msg_id):
+    async def set_eeprom_parameters(self, msg_id: int) -> None:
         """Save the parameter settings for the specified message.
 
         :param msg_id: The message ID of the message containing the parameters
             to be saved.
         """
-        payload = st.pack("<HH", 1, msg_id)
+        payload = st.pack("<HH", _CHANNEL, msg_id)
```

Comment:
```suggestion
        payload = st.pack("<HH", Tpz._CHANNEL, msg_id)
```
Use class constants.